instances.json

```diff
@@ -262,24 +262,9 @@
       "version": "v0.23.2"
     },
     {
-      "onion": "http://ojwp2gtj7dq7scd7gnbac6wp53tklgsicteabrnx2pr7zai64wriiaad.onion",
```

Comment:
Please recover the previous versions of the JSON and markdown files and rerun with

`./generate-instances-json.sh -i ./instances.txt -I ./instances.json -o ./instances.json`

so that the onion instances are inherited from the old file.

Reply:
@Daniel-Valentine Just uploaded the new commit with the changes requested.